modules/openy_features/openy_location/modules/openy_loc_filter/config/install/openy_loc_filter.location_filter_settings.yml

```diff
@@ -0,0 +1,8 @@
+locations:
```

Comment:
We should not provide a default config because each Open Y site has own branch NIDs.

Reply:
I think we should. But the module should support the empty set of locations.